README.md

```diff
@@ -161,107 +210,76 @@ Example HTML:
-</div>
-
-<div class="o-forms__message o-forms__message--error">
-	<p>This is a global error message that relates to the entire form</p>
 </div>
 ```
 
-[Messages examples](https://www.ft.com/__origami/service/build/v2/demos/o-forms/messages)
-
-#### Suffixes
-
-Suffixes are used to append content to an input, i.e. a button. The form control should be wrapped in a block-level element with a class of `.o-forms__affix-wrapper`. Suffixes (defined by `.o-forms__suffix`) can then be appended to this wrapper element.
+This can be used to highlight errors within a section of a form which contains multiple `.o-forms` fields.
 
 ```html
-<div class="o-forms">
-	<div class="o-forms__affix-wrapper">
-		<input type="text" class="o-forms__text" />
-		<div class="o-forms__suffix">
-			<button type="button" class="o-buttons o-buttons--secondary o-buttons--big">Go</button>
-		</div>
+<div class="o-forms-section o-forms-section--error">
+	<div class="o-forms-section__message">
+		<p>This is a section error message</p>
+	</div>
+	<div class="o-forms o-forms--error">
+		<label for="o-forms-section-error" class="o-forms__label">Field Label</label>
+		<input type="text" id="o-forms-section-error" placeholder="placeholder" class="o-forms__text" required />
+		<div class="o-forms__errortext">Invalid entry</div>
 	</div>
 </div>
 ```
 
-[Suffixes examples](https://www.ft.com/__origami/service/build/v2/demos/o-forms/suffix)
-
-#### "unskin" a form element
+Add `.o-forms-section--wide` to remove max width restrictions.
 
-Add the class `o-forms--unskin` to a text field to reset its appearance while keeping its vertical structure and other field properties (editable, focusable).
-
-```html
-<input type="text" class="o-forms--unskin" value="foo" />
-```
+[Section examples](https://www.ft.com/__origami/service/build/v2/demos/o-forms/sections)
 
-Properties of the `o-forms--unskin` class:
 
-- removes borders and rounded corners
-- removes the background color
-- removes the inner shadow added by some browsers
-- removes left-right spacing / padding
-- removes the "form field" appearance in supported browsers
-- editable
-- focusable
+#### Additional Features
 
+Additional features such as small text and select inputs are avalible. Please see [more examples on the Origami Registry](https://www.ft.com/__origami/service/build/v2/demos/o-forms/).
 
 ### Sass
 
 #### Silent mode
 
 In silent mode `o-forms` provides mixins for each set of form fields as well as some mixins to output basic form styles in one large chunk.
 
-#### oForms mixin
-
-The `oForms` mixin will output styles for the `o-forms` block, as well as all basic input styles supported by `o-forms` (`text`, `select`, `textarea`, `checkboxes` and `radio` buttons) as well as `labels`, valid and error states and, error text and helper text.
+The `oForms` mixin will output all features of `o-forms`. Turn off silent mode to output all `o-forms` features using this mixin automatically.
```

Comment:
should it be `oForms()`?

Reply:
Debatable.  Its only parameter is optional so you could `@include oForms` or  `@include oForms()`.